indico/web/client/styles/editor-output.scss

```diff
@@ -66,4 +66,9 @@
   code {
     background-color: #f5f5f5;
   }
+
+  ul,
+  ol {
+    margin-left: 1em;
```

Comment:
2em or 3em looks better IMHO

Reply:
How about this?

<img width="188" alt="image" src="https://github.com/indico/indico/assets/65413/bc2e5a44-dbff-47c6-b8f4-4f4d4cd64243">

Only the first level is indented 3em, the nested ones only 1.2em (aligns the text of the bullet of the nested list with the content of the parent item).

Reply:
Tho, not sure about this. When adding the initial indent for the top-level, but we have no text preceding it, we get something like this:

<img width="225" alt="image" src="https://github.com/indico/indico/assets/65413/05ad792b-9f3a-4252-8368-dc1cc8afc2a5">

Reply:
Alright. Then I'll push this.